### DISCUSSION TITLE
Simplify the usage sample

### DISCUSSION THREAD

**Discussion by roee88** [fybrik/fybrik#303] [Developers] [answered]
The Kubeflow notebook is too complex for being used a first sample guide. It requires too much and also depends on scripts that the user must actually understand. I propose to create a notebook sample that:

1. Provide an alternative to setting up an S3 account and bucket and uploading to it
2. Does not require Kubeflow installation but rather deploys a notebook as part of the guide (though the user can use Kubeflow if he wants to)
3. Uses the built-in catalog once we finalize integrating it
4. Describes how to add policies to OPA once we finalize OPA policies and dynamic management of them

For 1 we can leverage stuff from https://gist.github.com/roee88/0b24fa70930fed237242ac3f3fb4f5d5

For 2 we can use what we used in earlier stages of the project https://gist.github.com/roee88/9bb3606e8037e453714d4b9c413967e2

**Comment by roee88** [comment, marked as answer]
https://github.com/IBM/the-mesh-for-data/pull/455

**Reply by simanadler** [reply]
@roee88 Should we close this discussion?
